### DISCUSSION TITLE
Reminder for commit messages

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#188] [Show and tell]
Since we've begun or sprint of feature development,
let's remind ourselves to use [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) and PR names.
(We will most frequently use "Squash and Merge" so the commit messages in the branches will not be necessarily relevant but the PR title will be regarded as the commit message when merging into main.

An example PR title (hence the commit name) we've used with @hks1444:

chore(web): bootstrap the repo

chore: type of work
web: relevant app/subrepo
short and descriptive.

**Comment by Meminseeker** [comment]
For a quick reference, here is the basic definitions of the types:


**build:** Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
**ci:** Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
**docs:** Documentation only changes
**feat:** A new feature
**fix:** A bug fix
**perf:** A code change that improves performance
**refactor:** A code change that neither fixes a bug nor adds a feature
**style:** Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
**test:** Adding missing tests or correcting existing tests

**Reply by Meminseeker** [reply]
However, I couldn't see what `chore` meant. Can you explain it @umitcan07
